pkgs/tools/games/butler/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "butler";
+  version = "15.21.0";
+
+  src = fetchFromGitHub {
+    owner = "itchio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "14wcirxj5xcv60whvd7k8xcky0h2z8ym7gkhfb58vh3pfjcr5j5x";
```

```suggestion
    sha256 = "sha256-14wcirxj5xcv60whvd7k8xcky0h2z8ym7gkhfb58vh3pfjcr5j5x";
```
sri hashes must retain the prefix. sha256 might be switched to hash.